src/libraries/System.Private.CoreLib/src/System/IO/Path.Unix.cs

```diff
@@ -13,6 +13,32 @@ public static partial class Path
 
         public static char[] GetInvalidPathChars() => new char[] { '\0' };
 
+        // Checks if the given path is available for use.
+        public static bool Exists([NotNullWhen(true)] string? path)
+        {
+            try
+            {
+                if (path == null)
+                {
+                    return false;
+                }
+
+                if (path.Length == 0)
+                {
+                    return false;
+                }
+
+                ReadOnlySpan<char> fullPath = GetFullPath(path).AsSpan();
+                return Interop.Sys.LStat(fullPath, out Interop.Sys.FileStatus _) > 0;
```

Comment:
man page says lstat returns 0 on success. should this be `>= 0` (or ` == 0` possibly)?

Reply:
Yes I think I forgot to invert the if logic (it should be ```==0```)

Reply:
We should do what the existing code does.